src/iact_estimator/scripts/main.py

```diff
@@ -127,6 +140,109 @@ def main():
             seaborn_options = config["seaborn_options"]
             sns.set_theme(**seaborn_options)
 
+        # Basic observability checks
+        target_source = FixedTarget.from_name(source_name)
+        observer = Observer.at_site("Roque de los Muchachos")
+
+        crab = FixedTarget.from_name("Crab")
+
+        logger.debug("Defining observation constraints")
+        constraints = define_constraints(config)
+
+        start_datetime = (
+            Time(config["observation"]["start_datetime"])
+            if config["observation"]["start_datetime"] is not None
+            else Time(datetime.now(tz=observer.timezone))
+        )
+        year_days = get_days_in_this_year()
+        end_datetime = (
+            Time(config["observation"]["end_datetime"])
+            if config["observation"]["end_datetime"] is not None
+            else start_datetime + year_days
+        )
+        logger.info("Observation starts at %s", start_datetime)
+        logger.info("Observation ends at %s", end_datetime)
```

Comment:
I noticed that sometimes you use f-strings and here the simple format. I would use f-strings everywhere if possible, you can force a certain format also for f-strings if needed (I guess here you need it to print the Time object as string representation, but I would assume it should work also with f-strings).

Reply:
The reason is this
https://realpython.com/python-f-strings/#lazy-evaluation-in-logging